### ISSUE TITLE
Older release versions of OPM Flow

### ISSUE THREAD

**Issue by fdlberylian** [OPM/opm-simulators#2176]
Hi,

I am trying to compare the results from current release of Flow to its older versions (2018.10 and older). When I tried to build from source by cloning the 2018.10 branch, I got errors when "make"-ing it. I also tried downloading the 2018.10-final-release source code and got same errors when "make"-ing it. I wonder if there is a way I can get the 2018.10 release, maybe a pre-compiled version is available or something? Thanks in advance.

I also attach the error I got when building 2018.10 from source.
![Screenshot from 2019-11-21 14-37-25](https://user-images.githubusercontent.com/44946917/69342895-c9cedc80-0c6c-11ea-869d-cc3e3ae58890.png)

**Comment by joakim-hove** [comment]
For versions prior to the latest 2019.10 version you also have to download and build the libecl library : equinor/libecl on github.com

**Comment by fdlberylian** [comment]
I already have libecl installed and it still won't work. Is the version of libecl (or any prerequisites) have any effect on compiling the older versions? I have libecl ver 2.2 installed.

**Comment by akva2** [comment]
Yes. Use https://github.com/equinor/libecl/commits/2018.10 for 2018.10.

-------- Opprinnelig melding --------
Fra: fdlberylian <notifications@github.com>
Dato: 21.11.2019 16:08 (GMT+01:00)
Til: OPM/opm-simulators <opm-simulators@noreply.github.com>
Ko: Subscribed <subscribed@noreply.github.com>
Emne: Re: [OPM/opm-simulators] Older release versions of OPM Flow (#2176)


I already have libecl installed and it still won't work. Is the version of libecl (or any prerequisites) have any effect on compiling the older versions? I have libecl ver 2.2 installed.

**Comment by blattms** [comment]
@fdlberylian Cool. If you also check serial/parallel runtime for your models, I would be very interested in the data. Hopefully you are willing to share it, e.g. here.
